pages/api/patients/all.ts

```diff
@@ -0,0 +1,23 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'GET') {
+    try {
+      var patients = await prisma.patient.findMany({
+        include: {
+          user: true
+        }
+      })
+      res.status(200).json({ message: 'Success', patient: patients })
```

Comment:
We should not be assigning an array to `patient` you can pass `patients` directly and it will be parsed as 
```ts
{
...
patients: [ ...patients]
}
```
automatically

```suggestion
      res.status(200).json({ message: 'Success', patients })
```